src/addrman.cpp

```diff
@@ -962,6 +970,32 @@ std::optional<AddressPosition> AddrManImpl::FindAddressEntry_(const CAddress& ad
     }
 }
 
+size_t AddrManImpl::size_(std::optional<Network> net, std::optional<bool> in_new) const
+{
+    AssertLockHeld(cs);
+
+    if (!net.has_value()) {
+        if (!in_new.has_value()) {
+            return nNew + nTried;
+        } else if (in_new == true) {
+            return nNew;
+        } else {
+            return nTried;
+        }
+    }
+    if (auto it = map_counts.find(*net); it != map_counts.end()) {
+        auto entry = it->second;
+        if (!in_new.has_value()) {
+            return entry.first + entry.second;
+        } else if (in_new == true) {
```

Comment:
Here and above, maybe use `return (in_new == true) ? entry.first : entry.second;`?

Reply:
Changed it to the ternary operator.